src/monitoringRule/monitoringRuleFrequencies/types.ts

```diff
@@ -0,0 +1,7 @@
+import { MonitoringRulesFrequency } from '../monitoringRules/types';
+
+export type MonitoringRuleFrequencyRead = {
```

Comment:
could we put MonitoringRulesFrequency to this file that it is together MonitoringRuleFrequencyRead?

Reply:
discussed. let's keep as it is